### ISSUE TITLE
`man bun` gives `No manual entry for bun` on linux

### ISSUE THREAD

**Issue by csvenke** [oven-sh/bun#5575]
On linux when typing in terminal:
```
man bun
```
gives
```
No manual entry for bun
```

Not capitalizing on this pun opportunity must have been a huge oversight 😉

**Comment by dusda** [comment]
This is essential. For the joke alone.

**Comment by SpamixOfficial** [comment]
> This is essential. For the joke alone.

I agree. Though the manual page would also be appreciated since this is such a big project, with a lot of features.
